src/Holding/ViewHolding/HoldingBoundWith/useBoundWithHoldings.js

```diff
@@ -7,11 +7,11 @@ const useBoundWithHoldings = (boundWithItems) => {
   const [namespace] = useNamespace({ key: 'boundWithHoldings' });
 
   const holdingRecordIds = boundWithItems?.map(x => x.holdingsRecordId);
-  const queryIds = `(${holdingRecordIds.join(' or ')})`;
+  const queryIds = holdingRecordIds.join(' or ');
```

Comment:
My interpretation of this change is that remove the quotes changes the produced string to be empty when there are no Ids. And this will then evaluate to false (due to JS's crazy type coercion).

How wrong is my understanding of this change?

Reply:
@marcjohnson-kint yes, you are right the string will be empty but then the additional `(` `)` will be added creating `()` string. When that string is passed to `Boolean('()')` it will return `true` causing the `useQuery` to execute.

@maccabeelevine it's all good. I make many mistakes like that all the time too especially when refactoring/moving things around.